### PR TITLE
fix: add new "FOURC_*" env variable for setting log level

### PR DESCRIPTION
**Pull request by c0state**
Number prefixed environment variables aren't standard I believe. I ran into issues with this on my shell.

Adding new `FOURC_LOGGING_LEVEL` fallback env variable

### CODE REVIEW

**Comment by taion** [comment]
indeed:
> Environment variable names used by the utilities in the Shell and Utilities volume of IEEE Std 1003.1-2001 consist solely of uppercase letters, digits, and the '_' (underscore) from the characters defined in Portable Character Set and do not begin with a digit.

File: createLogging.js [hunk 1]
```diff
@@ -29,9 +29,17 @@ module.exports = function createLogging({
 } = {}) {
   const container = new Container();
   const TEST = Env.get('NODE_ENV', '') === 'test';
-  const useColor = Env.get.boolish('4C_LOGGING_USE_COLOR', true);
 
-  let LEVEL = Env.get('4C_LOGGING_LEVEL', 'info');
+  let useColor = Env.get.boolish('4C_LOGGING_USE_COLOR', null);
```
Comment (c0state) [inline createLogging.js]:
```suggestion
  let useColor = Env.get('4C_LOGGING_USE_COLOR', null);
```

**Comment by itajaja** [comment]
probably breaking?

**Comment by c0state** [comment]
> probably breaking?

Yeah trying to fix this as it's been sitting around. I think the logic is bad...

**Comment by c0state** [comment]
I think this should be good now.

**Comment by c0state** [comment]
Can I merge this or what's the process for this repo?

**Comment by taion** [comment]
@jquense can you review?

**Review by jquense — APPROVED** [review]

**Pull request merged**